rpcserver.go

```diff
@@ -344,6 +345,36 @@ func handleAddNode(s *rpcServer, cmd interface{}, closeChan <-chan struct{}) (in
 	return nil, nil
 }
 
+// handleNode handles node commands.
+func handleNode(s *rpcServer, cmd interface{}, closeChan <-chan struct{}) (interface{}, error) {
+	c := cmd.(*btcjson.NodeCmd)
+
+	addr := normalizeAddress(c.Addr, activeNetParams.DefaultPort)
+	var err error
+	switch c.SubCmd {
+	case "disconnect":
+		err = s.server.DisconnectNode(addr)
+	case "remove":
+		err = s.server.RemoveNode(addr)
+	case "connect":
+		switch *c.ConnectSubCmd {
+		case "perm", "non-perm":
+			err = s.server.ConnectNode(addr, *c.ConnectSubCmd == "perm")
+		default:
+			err = errors.New("invalid subcommand for node connect")
```

Comment:
I know that addnode does this too, but really it should be changed too.  The problem here is this will result in an internal error which is not accurate for an invalid parameter.  It really should be doing something like:

```Go
err = &btcjson.RPCError{
        Code: ErrRPCInvalidParameter,
        Message: "invalid subcommand for node connect",
    }
```